lib/router.js

```diff
@@ -124,6 +124,7 @@ module.exports = function (dataset, adapter) {
       adapter,
       'list',
       {
+        params: req.params,
```

Comment:
I'm thinking we might want to call this _routeParams_, just to be explicit about their nature.